addons/digest/models/digest.py

```diff
@@ -143,6 +143,7 @@ def _action_send_to_user(self, user, tips_count=1, consum_tips=True):
                 'kpi_data': self._compute_kpis(user.company_id, user),
                 'tips': self._compute_tips(user.company_id, user, tips_count=tips_count, consumed=consum_tips),
                 'preferences': self._compute_preferences(user.company_id, user),
+                'preserve_comments': True,
```

Comment:
Shouldn't it be ``self.env['mail.render.mixin'].with_context(preserve_comments=True)._render_template()`` ?

Reply:
Absolutely.